doc/man1/openssl-list.pod.in

```diff
@@ -73,6 +74,22 @@ displayed.
 In verbose mode, the algorithms provided by a provider will get additional
 information on what parameters each implementation supports.
 
+=item B<-serializers>
+
+Display a list of serializers.
+The serializers from a provider are displayed like this if the
+implementation is labeled with a single name:
+
+ foo @ bar (properties)
+
+or like this if it's labeled with multiple names:
+
+ { foo1, foo2 } @bar (properties)
+
+In both cases, C<bar> is the name of the provider and C<properties> are the
+property settings for the serializer.  The C<type> and C<format> properties
+distinguish between otherwise identical serializers.
+
```

Comment:
Why do you repeat here what's explained *generically* in "Display of algorithm names"?  The following should be enough here:

```
=item B<-serializers>

Display a list of serializers.
See L</Display of algorithm names> for a description of how names are
displayed.

In verbose mode, the algorithms provided by a provider will get additional
information on what parameters each implementation supports.
```
```

Reply:
Changed.